### ISSUE TITLE
Mainnet 100% staking on GIVpower confusing numbers

### ISSUE THREAD

**Issue by aminlatifi** [Giveth/giveth-dapps-v2#1346]
Currently, 100% GIV staking card on mainnet/kovan is integrated with the gnoso-chain one which is GIVpower. The values in it are confusing for me and I didn't understand at first sight what's going on till Cherick told me. 
My staked amount was receiving lower reward than the APR, the unstake and harvest are haversting the mainnet farm and stake will redirect me to the gnosis-chain one.

I suggest having different cards for them! one for presenting the 100% giv staking on mainnet and another to redirect user to GIVpoewr on gnosis-chain
<img width="582" alt="image" src="https://user-images.githubusercontent.com/5684607/184122401-3a517881-694f-4c73-ac5d-be430951f045.png">

**Comment by aminlatifi** [comment]
@laurenluz What do you think?

**Comment by laurenluz** [comment]
Hey @aminlatifi ! We plan to end the GIV staking on mainnet when we launch GIVpower, so this won't be an issue. I think it's just a weird issue w/ our test deployment that has both.

Actually, the mainnet GIV staking will end (most likely) next week, and will not be extended, according to [this forum post](https://forum.giveth.io/t/extending-the-givfarm-again/572/24)

**Comment by aminlatifi** [comment]
> Hey @aminlatifi ! We plan to end the GIV staking on mainnet when we launch GIVpower, so this won't be an issue. I think it's just a weird issue w/ our test deployment that has both.
> 
> Actually, the mainnet GIV staking will end (most likely) next week, and will not be extended, according to [this forum post](https://forum.giveth.io/t/extending-the-givfarm-again/572/24)

So it makes sense more to have a separate card for that! because removing/archiving 100% giv staking on mainnet may cause later issues. If we have totally different cards on mainnet (one for 100% GIV farming on mainnet and another for givpower) we will be safer in the future.

**Comment by MohammadPCh** [comment]
@aminlatifi @laurenluz  In the new design where we show the cards of both networks together, it is better to show the mainnet give farm card in the old way and when the user clicks on the stake, show the bridge modal to him. This design can be tested now at this address:
https://givpower.vercel.app/givfarm

**Comment by franculio** [comment]
Tested on https://givpower.vercel.app/givfarm

I found that the two cards are separated. The only thing I can click on the mainnet givfarm is "provide liquidity" and it links me to an empty site. Is this what the issue was about?

<img width="1344" alt="Captura de Pantalla 2022-09-16 a las 18 33 48" src="https://user-images.githubusercontent.com/79286405/190687750-84d1749f-d9f0-425c-9a63-d61aeb677934.png">
